pkgs/tools/security/zlint/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, testers
+, zlint
+}:
+
+buildGoModule rec {
+  pname = "zlint";
+  version = "3.4.0";
+
+  src = fetchFromGitHub {
+    owner = "zmap";
+    repo = "zlint";
+    rev = "v${version}";
+    hash = "sha256-l39GdfEKUAw5DQNjx6ZBgfGtengRlUUasm0G07kAA2A=";
+  };
+
+  modRoot = "v3";
+
+  vendorHash = "sha256-OiHEyMHuSiWDB/1YRvAhErb1h/rFfXXVcagcP386doc=";
+
+  postPatch = ''
+    # Remove a package which is not declared in go.mod.
+    rm -rf v3/cmd/genTestCerts
+  '';
+
+  subPackages = [
+    "cmd/zlint"
+    "cmd/zlint-gtld-update"
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
+  # Checks rely on .git directory, leaveDotGit makes the source derivation flaky.
+  doCheck = false;
```

Comment:
Can we patch that it requires a git directory?

Reply:
that got dropped in 3.4.1 which just released on Sunday and that I missed.
https://github.com/NixOS/nixpkgs/pull/202590